app/models/message.rb

```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
```

Comment:
- [ ] Please make sure that the app works properly. The following errors are shown when the app starts, to replicate this please clone the apps again and run them individually. These are mostly packages missing errors which should be easy to fix.

![image](https://github.com/katarighe/hello-rails-back-end/assets/62290545/801c716f-bd45-471e-898a-78a454129598)

Reply:
✅ Solved